sensehat/README.md

```diff
@@ -54,6 +54,17 @@ display.draw(bitmap);
 // Close the display when done.
 display.close();
 ```
+```
+                try (BaroTemp bt = SenseHat.openBaroTemp()) {
```

Comment:
I think you're indenting too much here.
Your snippet also lacks a comment of what it does.

Reply:
Fixed. And thanks for the awesome code review!